Makefile

```diff
@@ -7,6 +7,16 @@ RDMD_FLAGS=--force --build-only --compiler=$(D_COMPILER) $(D_FLAGS)
 
 PLATFORM := $(shell uname -s)
 
+LDMD=ldmd2
+
+# force this to run on every make. make eats newlines.
+LDC_VERSION_STRING := $(shell $(LDMD) --version | head -1)
+LDC_DMD_LLVM_VERSION_STRING := $(shell $(LDMD) --version | head -2 | tail -1)
+LDC_LDC_VERSION_STRING := $(shell $(LDMD) --version | head -3 | tail -1)
```

Comment:
I've had so many build-chain-dependent issues (especially with debuggers, but also with e.g. exception unwinding) that I thought it best to include as much as possible. I wouldn't call it of interest exactly, but I think it would be good to be able to surface this easily. What about having all of this in `sambamba --version` and keeping usage otherwise clean?